### ISSUE TITLE
feat: edit HTML source

### ISSUE THREAD

**Issue by ShutdownRepo** [GitbookIO/integrations#183]
Is there any way we could use integrations to edit the source code and add, for example, blocks in `<head></head>`, add `<script>` tags etc. ?

**Comment by addisonschultz** [comment]
Hi @ShutdownRepo, 

This is a great feature request - but unfortunately isn't possible at the moment. We don't allow integrations to inject custom JS or CSS, for security reasons. We do have a scope that allows this to be possible for GitBook built and verified integrations at the moment - so for now I'll put this request for us to have a look at in the future.

**Comment by ShutdownRepo** [comment]
Fair enough. Is there any way an official Google AdSense integration could be made then? Happy to help make it if needed

**Comment by addisonschultz** [comment]
Hi @ShutdownRepo, 

Unfortunately right now the scope required for creating an integration like this is closed for the initial launch of the integration platform. That being said, we do know it's a highly requested feature, so have this high up on our radar for the next iteration(s) of the platform in general. We need to define a few things and figure out a secure and sustainable way to manage these kinds of scopes - and until we have that in place we'll do our best to monitor the requests coming in!
